gradle/mockito-core/javadoc.gradle

```diff
@@ -52,6 +52,23 @@ javadoc {
     options.addStringOption('Xwerror', '-quiet')
     options.addBooleanOption('-allow-script-in-comments', true)
 
+    // attempt to fix https://github.com/mockito/mockito/issues/2765
+    doFirst {
+        download.run {
+            src 'https://docs.oracle.com/javase/6/docs/api/package-list'
+            // somehow downloading src/javadoc or buildDir/javadoc didn't work, probably something related to the folder
```

Comment:
Nit: replace comment with the following:

```java
// Since the JavaDoc output is generated in `$buildDir/javadoc`, specify a different location for this task
```